internal/report/report.go

```diff
@@ -49,8 +49,8 @@ func NewWriter(cfg config.ReportConfig) (Writer, error) {
 
 	w := os.Stdout
 	isStdout := true
-	if cfg.OutputFile != "" {
-		f, err := os.Create(cfg.OutputFile)
+	if config.Get(cfg.OutputFile) != "" {
+		f, err := os.Create(config.Get(cfg.OutputFile))
```

Comment:
You can avoid one call to `config.Get` with:

```suggestion
	if outputFile := config.Get(cfg.OutputFile); outputFile != "" {
		f, err := os.Create(outputFile)
```